receiver/activedirectorydsreceiver/counters.go

```diff
@@ -24,9 +24,9 @@ const (
 	draOutboundObjects                     = "DRA Outbound Objects/sec"
 	draInboundProperties                   = "DRA Inbound Properties Total/sec"
 	draOutboundProperties                  = "DRA Outbound Properties/sec"
-	draInboundValuesDNs                    = "DRA Inbound Values (DNs only)/sec"
+	draInboundValuesDNs                    = "DRA Inbound Values (DNs only)/sec" //revive:disable-line:var-naming
```

Comment:
Can you share why the variable naming here is raising linting errors?

Reply:
Lint thinks is a typo of Domain Name System, so it wants an uppercase `s` however here the meaning is a Distinguished Name(s): https://learn.microsoft.com/en-us/previous-versions/windows/desktop/ldap/distinguished-names

Reply:
Perfect, thanks for explaining!